docs/sources/developers/plugins/add-authentication-for-data-source-plugins.md

```diff
@@ -348,3 +348,31 @@ func (ds *dataSource) CallResource(ctx context.Context, req *backend.CallResourc
 ```
 
 > **Note:** Due to a bug in Grafana, using this feature with PostgreSQL can cause a deadlock. For more information, refer to [Grafana causes deadlocks in PostgreSQL, while trying to refresh users token](https://github.com/grafana/grafana/issues/20515).
+
+## Forward cookies for the logged-in user
+
+Your data source plugin can forward certain cookies for the logged-in Grafana user to the data source.
+
+To allow Grafana to pass cookies to the plugin, update the data source configuration and set the` jsonData.keepCookies` property to an array of strings, where the strings are the names of the cookies that should be passed to the plugin. The [DataSourceHttpSettings](https://developers.grafana.com/ui/latest/index.html?path=/story/data-source-datasourcehttpsettings--basic) provides a UI element, the **Allowed cookies** option, for this. You can also build an appropriate input element to set `jsonData.keepCookies` in your data source configuration page UI.
```

Comment:
Can you please rewrite this to focus on the https://developers.grafana.com/ui/latest/index.html?path=/story/data-source-datasourcehttpsettings--basic as the only/preferred way to use this feature, maybe not mentioning the `jsonData.keepCookies`?

Reply:
sure, text changed.